src/System.Management.Automation/resources/CommandBaseStrings.resx

```diff
@@ -156,6 +156,9 @@
   <data name="PauseHelpMessage" xml:space="preserve">
     <value>Pause the current pipeline and return to the command prompt. Type "{0}" to resume the pipeline.</value>
   </data>
+  <data name="ProgramFailedToComplete" xml:space="preserve">
```

Comment:
There is a ProgramFailedToExecute resource string in `ParserStrings.resx` but that doesn't seem to be quite right for this runtime error.  Is `CommandBaseStrings.resx` the correct file for this resource sting?

Reply:
It seems it is not used at all.

Reply:
It is used in the same NativeCommandProcessor.cs file during the creation of another exception `ApplicationFailedException` that I thiink covers the general case of native command execution failure.  This does make me wonder if someone looking at the code in the future is going to wonder why there's both an `ApplicationFailedException` and a `NativeCommandException`.

Reply:
There is still used ParserStrings.CantActivateDocumentInPowerShellCore with InterpreterError.NewInterpreterException.

Reply:
> This does make me wonder if someone looking at the code in the future is going to wonder why there's both an ApplicationFailedException and a NativeCommandException.

Worth adding a comment in the elements added in this PR to note that despite those things pre-existing, we added new ones to be specifically for the native error action preference feature to ensure it didn't accidentally clobber any existing concepts.

Reply:
Adding this in my PR

Reply:
I think we need to open new issue in Docs repo to explain `ApplicationFailedException` (PowerShell can not run normally an non-backgroud application) vs `NativeCommandException` (PowerShell could be able to run non-backgraund application but throw due to non zero return code).